iset.mm

```diff
@@ -287,9 +287,9 @@ described in the Metamath book (or the online HELP).
 or a new relationship among symbols that can be eliminated; they always
 extend the definition of a wff or class.  Metamath blindly treats $a
 statements as new given facts but does not try to justify them.  The
-mmj2 program will justify the definitions as sound, except for 5 (df-bi,
-df-cleq, df-clel, df-clab, df-sbc) that require a more complex metalogical
-justification by hand.
+mmj2 program will justify the definitions as sound, except for five of them
+(df-bi, df-cleq, df-clel, df-clab, df-sbc) that require a more complex
+metalogical justification.
```

Comment:
It doesn't seem like df-sbc is an exception:

https://github.com/metamath/set.mm/blob/e50dd75c9a8fb079cc6dbb3620b787e247699561/.github/workflows/verifiers.yml#L289

but I'm probably missing something

Reply:
Actually, I blindy copied the analogous part from set.mm. So one of them has to be fixed. Maybe I "fixed" the wrong one... I'm going to check.

Reply:
Done: you were right, actually iset.mm was correct and it is set.mm that had to be fixed. (Well, the correct fix would be to simply have set.mm and iset.mm share that information, by including a common file. That's still in my plans, for later.)